app/components/users/library/library-grid/library-entry.js

```diff
@@ -11,6 +11,11 @@ export default Component.extend({
   tagName: '',
   readOnlyModal: false,
 
+  hasReaction: computed('libraryEntry.mediaReaction', function() {
+    const reaction = get(this, 'libraryEntry.mediaReaction');
```

Comment:
Should change this to

`belongsTo('mediaReaction').value();`

so it really is a read-only check.